src/runtime/locale/cs.ts

```diff
@@ -0,0 +1,20 @@
+import { LocaleStrings } from '../types'
```

Comment:
The file name should be `cz` as well, shouldn't it?

Reply:
I think "cs" is the correct form. The "cz" is for the name of the country, not the language. See e.g. language settings for CKEditor `config.language = 'cs';`

Reply:
Following the ISO spec, it seems to be https://en.wikipedia.org/wiki/ISO_3166-2:CZ

Reply:
I'll leave the decision up to you, but the ISO you posted is about the country, not the language. Look at ISO 639-1, plus the generally accepted language codes are e.g. en-US or en-UK (en=language US=region). Czech Republic has cs-CZ, Slovakia has sk-SK.

Reply:
Thank you for that information! Then, we can leave it as is.